shared/actions/chat.js

```diff
@@ -664,6 +825,8 @@ function * chatSaga (): SagaGenerator<any, any> {
     safeTakeEvery(Constants.postMessage, _postMessage),
     safeTakeEvery(Constants.startConversation, _startConversation),
     safeTakeEvery(Constants.updateMetadata, _updateMetadata),
+    safeTakeEvery(Constants.clickedAttach, _clickedAttach),
+    safeTakeEvery('chat:loadAttachment', _loadAttachment),
```

Comment:
There is no `Constants.loadAttachment`, I don't think those are useful at all and are only boilerplate